python/src/PythonGreyscale.cc

```diff
@@ -0,0 +1,47 @@
+#include <boost/python/numpy.hpp>
+#include <iostream>
```

Comment:
Project then dependencies then system includes (I think this is in Google's C++ style guide)